stable/redis/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 3.2.4
+version: 3.2.5
```

Comment:
I think you should increase the major version since this change breaks the compatibility with previous versions.

Reply:
Ok, I'll bump it up a major version.